### PR TITLE
Document Huawei Cloud OBS support

### LINKED ISSUE #1614: Document: feat(object store): support obs
This issue tracks the documentation update needed for the merged PR #13844.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13844
Source PR Merged At: 2023-12-07T06:10:48Z

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - Still waiting for the inputs from the Operator side. Tracking: https://github.com/risingwavelabs/risingwave-operator/issues/557

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/13844
  - https://github.com/risingwavelabs/risingwave/pull/13853

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1614

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by ShanlanLi — APPROVED** [review]
LGTM

File: docs/deploy/risingwave-docker-compose.md [hunk 1]
```diff
@@ -132,6 +131,22 @@ Run the following command to start a RisingWave cluster:
 docker compose -f docker-compose-with-hdfs.yml up
 ```
 
+### Huawei Cloud OBS
+
+To use Huawei Cloud OBS as the storage backend, you need to configure your OBS credential information in the `multiple_object_storage.env` file. Uncomment and set the following environment variables: `OBS_ENDPOINT`, `OBS_ACCESS_KEY_ID`, and `OBS_SECRET_ACCESS_KEY`. Don't include the bucket name in the endpoint.
+
+In the `docker-compose-with-obs.yml` file, specify the bucket name via the `hummock+obs` parameter. Replace `<bucket-name>` with the name of your OBS bucket.
```
Comment (neverchanje) [inline docs/deploy/risingwave-docker-compose.md:138]: A bit weird to see that we only document how to configure docker compose, while k8s is what we recommend for production. We may need to add links somewhere, referring to the config update doc for k8s

**Review by neverchanje — APPROVED** [review]
Rest LGTM

File: docs/deploy/risingwave-docker-compose.md (hunk 1 above)
Comment (CharlieSYH) [reply docs/deploy/risingwave-docker-compose.md:138]: We didn't mention certain storage services on the K8s page. We can include them when we have the ZH version of the docs.

**Pull request merged**